addons/base_address_extended/models/__init__.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import res_company
-from . import res_country
 from . import res_partner
+from . import res_city
+from . import res_country
```

Comment:
why move res_country? unnecessary change to the diff. I also usually prefer alphabetical order

Reply:
Alphabetical order, the module is under heavy changes, I vote for it.

Reply:
How is that alphabetical order changes? The company has no formatting anymore in the Form view.